budgetApp1/budgetApp1/ProfileViewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ProfileViewController.swift
+//  budgetApp1
+//
+//  Created by Lehlohonolo Nkadimeng on 2024/05/29.
+//
+
+import UIKit
+
+class ProfileViewController: UIViewController {
+
+    // MARK: - Variables
+    var username: String?
+    var loggedInStatus = true
+    private lazy var viewModel = ProfileViewModel()
+    
+    // MARK: - IBOutlets
+    @IBOutlet private weak var usernameLabel: UILabel!
+    @IBOutlet private weak var profileImage: UIImageView!
+    
+    // MARK: - Functions
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        customizeImage()
+        
+        if let loggedInUser = viewModel.getLoggedInUser(), let username = loggedInUser.username {
```

Comment:
why do you need the loggedInUser and the username

Reply:
and please make this a private func and call it in the viewDidLoad. We don't want the viewDidLoad to get messy so rather put it in a function and cll the function in here

Reply:
I am using this to get the username of the user who is loggedIn so I can show it in the profile screen.

Reply:
i see, why not just create a function that returns the username straight from the get go instead of returning the whole user?